### PR TITLE
Fix to Issues with timestamp formatting in the history

### PR DESCRIPTION
**Pull request by PrajwalM2212**
## Description

Provides a fix to Issue with timestamp formatting in the history

## Related issues and discussion
#939

### CODE REVIEW

**Comment by PrajwalM2212** [comment]
@teolemon  I have changed the function for timestamp formatting.

**Comment by Karljoones** [comment]
Can we have a screenshot of the changes? Or do the date formats come out the same?

**Comment by PrajwalM2212** [comment]
@Karljoones  It comes out the same.

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/views/adapters/HistoryListAdapter.java [hunk 1]
```diff
@@ -48,7 +48,7 @@ public void onBindViewHolder(HistoryScanHolder holder, int position) {
         Date date = list.get(position).getTime();
 
         long time = date.getTime();
-        String timeStamp = (String)DateUtils.getRelativeTimeSpanString(time);
+        String timeStamp = String.valueOf(DateUtils.getRelativeTimeSpanString(time,System.currentTimeMillis(), DateUtils.HOUR_IN_MILLIS,DateUtils.FORMAT_ABBREV_RELATIVE));
```
Comment (policorn) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/views/adapters/HistoryListAdapter.java]: Timestamps within the last hour are rounded to "0 hr. ago". I personally prefer showing "X min. ago" for those cases and "A moment ago" or "Just now" for timestamps within the last minute.

Comment (PrajwalM2212) [reply app/src/main/java/openfoodfacts/github/scrachx/openfood/views/adapters/HistoryListAdapter.java]: [DateUtils]( https://developer.android.com/reference/android/text/format/DateUtils.html) does not provide that functionality.

Comment (PrajwalM2212) [reply app/src/main/java/openfoodfacts/github/scrachx/openfood/views/adapters/HistoryListAdapter.java]: We may have to use some library.

**Comment by PrajwalM2212** [comment]
fix_time

@teolemon @Karljoones @policorn

**Comment by policorn** [comment]
Thanks @PrajwalM2212! Merged

**Pull request merged**